easybuild/easyconfigs/c/Cgl/Cgl-0.60.8-foss-2023b.eb

```diff
@@ -0,0 +1,50 @@
+easyblock = "ConfigureMake"
+
+name = 'Cgl'
+version = '0.60.8'
+
+homepage = "https://github.com/coin-or/Cgl"
+description = """The COIN-OR Cut Generation Library (Cgl) is a collection of cut generators that
+can be used with other COIN-OR packages that make use of cuts, such as, among
+others, the linear solver Clp or the mixed integer linear programming solvers
+Cbc or BCP. Cgl uses the abstract class OsiSolverInterface (see Osi) to use or
+communicate with a solver. It does not directly call a solver."""
+
+toolchain = {'name': 'foss', 'version': '2023b'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = ['https://github.com/coin-or/Cgl/archive/refs/tags/releases']
+sources = ['%(version)s.tar.gz']
+checksums = ['1482ba38afb783d124df8d5392337f79fdd507716e9f1fb6b98fc090acd1ad96']
+
+builddependencies = [
+    ('Autotools', '20220317'),
+    ('Doxygen', '1.9.8'),
+    ('pkgconf', '2.0.3'),
+]
+
+dependencies = [
+    ('CoinUtils', '2.11.10'),
+    ('Osi', '0.108.9'),
+    ('Clp', '1.17.9'),
+]
+
+# Use CoinUtils from EB
+configopts = '--with-coinutils-lib="-lCoinUtils" '
+configopts += '--with-coinutils-datadir=$EBROOTCOINUTILS/share/coin/Data'
+# Use Clp from EB
+configopts += '--with-clp-lib="-lOsiClp -lClpSolver -lClp" '
+configopts += '--with-clp-datadir=$EBROOTCLP/share/coin/Data'
+# Use Osi from EB (also needs links to Clp due to OsiClpSolver)
+configopts += '--with-osi-lib="-lOsiClp -lClpSolver -lClp -lOsi" '
+configopts += '--with-osi-datadir=$EBROOTOSI/share/coin/Data '
```

Comment:
It's less error prone (no risk of missing spaces) to use
```
configopts = ' '.join([
    '--with-coinutils-lib="-lCounUtils"',
   '...',
])
```
I advise changing to that format.

Reply:
PR updated with  to use join